Tests/Tests/Core/Extensions/Other/XCTestCaseTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  XCTestCaseTests.swift
+//  PovioKit_Tests
+//
+//  Created by Borut Tomazin on 15/09/2022.
+//  Copyright © 2022 Povio Inc. All rights reserved.
+//
+
+import XCTest
+import PovioKit
+
+class XCTestCaseTests: XCTestCase {
+  func testEqualImage() {
+    XCTAssertEqualImage(UIImage(systemName: "sun.min"), UIImage(systemName: "sun.min"))
+    XCTAssertEqualImage(UIImage(), UIImage())
+  }
+  
+  func testNotEqualImage() {
+    XCTAssertNotEqualImage(UIImage(systemName: "sun.min"), UIImage(systemName: "sun.max"))
+    XCTAssertNotEqualImage(UIImage(), UIImage())
+  }
+  
+  func testEqualFont() {
+    XCTAssertEqualFont(lhs: UIFont.systemFont(ofSize: 12), rhs: UIFont.systemFont(ofSize: 12))
+  }
+  
+  func testNotEqualFont() {
+    XCTAssertNotEqualFont(lhs: UIFont.systemFont(ofSize: 12), rhs: UIFont.systemFont(ofSize: 13))
+    XCTAssertNotEqualFont(lhs: UIFont.systemFont(ofSize: 12), rhs: UIFont.boldSystemFont(ofSize: 12))
+    XCTAssertNotEqualFont(lhs: UIFont.systemFont(ofSize: 12), rhs: UIFont.italicSystemFont(ofSize: 12))
+  }
+}
```

Comment:
I would add a new assertion for each test where we have optional values as arguments, that way we can further assert that we are covering all cases.

For example :
``` swift
XCTAssertNotEqualImage(UIImage(), nil)
XCTAssertNotEqualFont(lhs: UIFont.systemFont(ofSize: 12), rhs: nil)
```

Reply:
Yes